_posts/1-4-1-CLI.md

```diff
@@ -6,123 +6,322 @@ section: Extensions
 ---
 
 # Shoutem CLI
-Shoutem Command Line Interface (CLI) is a tool to manage your extensions. By using it you can create extensions, generate code snippets, upload extensions to your Shoutem account and install them on your applications. Here you can find a list of available commands. 
+Shoutem Command Line Interface (CLI) is a tool that helps you build extensions. Using it, you can create extensions, generate code snippets, upload extensions to your Shoutem account and install them on your applications.
```

Comment:
'CLI allows you to create extensions, ...' or 'It allows you to...'
This sounds like a Croatian sentence :)